core/src/main/java/org/elasticsearch/index/engine/ShadowEngine.java

```diff
@@ -190,8 +189,6 @@ public void refresh(String source) throws EngineException {
         try (ReleasableLock lock = readLock.acquire()) {
             ensureOpen();
             searcherManager.maybeRefreshBlocking();
-        } catch (AlreadyClosedException e) {
```

Comment:
Same comment here: do we still need to catch this and then immediately rethrow otherwise it will get eaten by the general catch block below?